tasks/sep/004-fpac-upgrade/README.md

```diff
@@ -0,0 +1,56 @@
+# Sepolia Devnet 0 FPAC Upgrade
+
+## Objective
+
+This is the playbook for executing the Fault Proof Alpha Chad upgrade on Sepolia devnet 0.
```

Comment:
Worth clarifying that this is the runbook for the prod sepolia testnet, not the devnet-0. Multiple places.